### DISCUSSION TITLE
Enhancement: For the text-container use px instead of rem

### DISCUSSION THREAD

**Discussion by Tomato6966** [danny-avila/LibreChat#4773] [Feature Requests & Suggestions]
### What features would you like to see added?

because you use rem for the max width of the container, when you scroll out the text goes smaller, but also the container
i'd like it when the container would stay in the same width but only the text would go smaller

This would make it easier for screenshots, hence more text could be displayed within the container when scrolling out

### More details

See the feature.

In the picture you can see how it looks in my browser on default, because on default i have every page zoom on -50% due to my screen.

### Pictures

![image](https://github.com/user-attachments/assets/e05a3267-84ab-4af4-b2c5-a62de5447451)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
PRs welcome. you can also increase the chat font size if needing to zoom out like this

![image](https://github.com/user-attachments/assets/0e5120ff-afe0-47a4-8391-89fd5e9fa08e)

**Reply by Tomato6966** [reply]
It's not about font size, it's about the container width, since you use rem instead of px it's getting smaller in widht, tho it'S not requied to get smaller in widht, my opinion only

**Reply by evtn** [reply]
when you zoom out both `px` and `rem` get smaller, `rem` size is fixed in pixels in your browser settings (by default `1rem=16px`).

you can install Stylus extension (or any other userstyle manager) and redefine styles for any element using `vw` units (that way it will not shrink with zoom)
